src/sage/matroids/advanced.py

```diff
@@ -63,6 +64,4 @@
 from . import lean_matrix
 from .extension import LinearSubclasses, MatroidExtensions
 from .union_matroid import MatroidUnion, MatroidSum, PartitionMatroid
-
-from sage.misc.lazy_import import lazy_import
-lazy_import('sage.matroids.graphic_matroid', 'GraphicMatroid')
+from .graphic_matroid import GraphicMatroid
```

Comment:
I made it a `lazy_import` for modularization purposes: `sage.matroids` is intended to be shipped as part of the distribution **sagemath-modules**; graph functionality becomes available at runtime if the distribution **sagemath-graphs** is installed

Reply:
I changed this when I cythonized `GraphicMatroid`. It doesn't compile.

Reply:
It works just fine actually. Can I run tests as if `graphs` isn't available?